packages/react-router/lib/dom/ssr/data.ts

```diff
@@ -5,16 +5,6 @@ import "../global";
  */
 export type AppData = unknown;
 
-export function isResponse(value: any): value is Response {
-  return (
-    value != null &&
-    typeof value.status === "number" &&
-    typeof value.statusText === "string" &&
-    typeof value.headers === "object" &&
-    typeof value.body !== "undefined"
-  );
-}
```

Comment:
We had this duped in `@remix-run/router` and `remix-run/server-runtime` so we can share the router one now